app/models/post.rb

```diff
@@ -3,6 +3,11 @@ class Post < ApplicationRecord
   has_many :likes, foreign_key: 'post_id'
   has_many :comments, foreign_key: 'post_id'
 
+  validates :title, presence: true
+  validates :text, length: { maximum: 250, minimum: 2 }, presence: true
+  validates :comments_counter, comparison: { greater_than_or_equal_to: 0 }, presence: true
+  validates :likes_counter, comparison: { greater_than_or_equal_to: 0 }, presence: true
```

Comment:
- [ ] The `comparison` validation helper doesn't ensure the value is numerical. You can use [`numericality`](https://guides.rubyonrails.org/active_record_validations.html#numericality) validation helper instead of it.


> LikesCounter must be an **integer** greater than or equal to zero.

Reply:
Done! 👍